### PR TITLE
[RG-396] Move help to txt files

### PR DESCRIPTION
**Pull request by KochynVolodymyr**
### Motivate of the pull request
 - [ ] To address an existing issue. If so, please provide a link to the issue: <issue id>
 - [x] Breaking new feature. If so, please describe details in the description part.

 ### Describe the technical details
Move help materials to txt files. 

 ### Which part of the code base require a change
 
 - [x] Library: compiler, foedagcore
 - [ ] Plug-in: <Specify the plugin name>
 - [ ] Engine
 - [ ] Documentation
 - [ ] Regression tests
 - [ ] Continous Integration (CI) scripts

### CODE REVIEW

**Comment by KochynVolodymyr** [comment]
@ravic-rs it's ready for merge but something wrong with commits.

File: etc/help.txt [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+   set_property mode <io_mode_name> <device_io_name> : Constraints pin mode (Use in constraint.pin file)
+
+----------
+--- IP ---
+----------
+   add_litex_ip_catalog <directory> : Browses directory for LiteX IP generators, adds the IP(s) to the IP Catalog
+   ip_catalog ?<ip_name>?     : Lists all available IPs, and their parameters if <ip_name> is given 
+   ip_configure <IP_NAME> -mod_name <name> -out_file <filename> -version <ver_name> -P<param>="<value>"...
+                              : Configures an IP <IP_NAME> and generates the corresponding file with module name
+   ipgenerate ?clean?         : Generates all IP instances set by ip_configure
```
Comment (ravic-rs) [inline etc/help.txt:91]: For all clean commands: Deletes files generated from this task.

**Review by ravic-rs — COMMENTED** [review]
Please also add the program device help from https://github.com/os-fpga/FOEDAG/pull/1087
---------------
--- Program ---
---------------
  program_device              : Program the device via JTAG 
       -b bitstream           : Target bitstream file for loading
	   -c config_file         : Configuration file
	   -n index               : Index of device in JTAG chain

**Comment by alaindargelas** [comment]
@ravic-rs @KochynVolodymyr , let's get this merged.

File: etc/help.txt (hunk 1 above)
Comment (ravic-rs) [reply etc/help.txt:91]: Some clean commands needs to be updated.

**Comment by ravic-rs** [comment]
> Please also add the program device help from #1087
> 
> ## --- Program ---
> program_device : Program the device via JTAG -b bitstream : Target bitstream file for loading -c config_file : Configuration file -n index : Index of device in JTAG chain

This one is done but I broke the build.

**Pull request merged**